public/app/AppWrapper.tsx

```diff
@@ -123,3 +156,13 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
     );
   }
 }
+
+const getStyles = (theme: GrafanaTheme2, navPosition: typeof contextSrv.user.navPosition) => ({
+  grafanaApp: css`
+    @media ${styleMixins.mediaUp(`${theme.breakpoints.values.md}px`)} {
```

Comment:
the new theme has this built in: https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/themes/GlobalStyles/card.ts#L127

Reply:
ah good to know, there's probably a few more places i can fix then